### ISSUE TITLE
Trakt Configuration tool won't login to Plex account

### ISSUE THREAD

**Issue by montizzl3** [trakt-for-plex/configuration#20]
I have latest PMS and Trakt Plugin 1.0.3.3.  I have multiple accounts created in the Trakt Configuration tool.  They are all scrobbling and syncing just fine.  I tried to add another account and it won't login to the Plex.tv account.  When I enter the plex username and password and save it it just goes back showing the PIN.  If I sign in using PIN, it just goes back to a new PIN.  Endless loop.  It has only done this with two different Plex accounts.  I have 5 other accounts linking trakt accounts to plex accounts with no issues.  Any idea why certain plex accounts won't login in the Trakt for Plex Configuration tool?

**Comment by fuzeman** [comment]
Could you provide some more details on this issue by doing the following:

1. Navigate to the configuration site
2. Open the browser console *(Ctrl + Shift + J on Chrome, Ctrl + Shift + K on Firefox)*
3. Attempt another login with one of the Plex accounts that aren't connecting correctly
4. Screenshot the "Console" and "Network" tabs, and either attach the images to a comment on this issue or email them to me@dgardiner.net

**Comment by montizzl3** [comment]
<img width="657" alt="screen shot 2016-07-19 at 12 30 42 am" src="https://cloud.githubusercontent.com/assets/16807269/16938670/71feb0ac-4d48-11e6-90b5-fe7183c1d43e.png">
<img width="884" alt="screen shot 2016-07-19 at 12 32 23 am" src="https://cloud.githubusercontent.com/assets/16807269/16938672/754257c8-4d48-11e6-893f-2f10885bc336.png">

**Comment by fuzeman** [comment]
Are you waiting for the configuration site to complete the PIN authentication process? it should automatically switch from the PIN code to a view displaying your account avatar and name.

-----------------------

**For example:**

1. PIN code is displayed

    ![](https://cloud.githubusercontent.com/assets/526664/16939164/097b06b2-4dd3-11e6-8cb0-d8d421c7787a.png)

2. PIN code is entered on https://plex.tv/pin
3. Configuration site detects the PIN was entered *(this can take a few seconds)*, and displays your account details

    ![](https://cloud.githubusercontent.com/assets/526664/16939188/5b7f25c4-4dd3-11e6-85ad-cf10ed747269.png)

4. The account can now be saved with the Plex account authorization

**Comment by montizzl3** [comment]
I changed the mode to Basic and used my username and password. After
clicking login the gravatar came up like it was logged into the Plex
account but there was not the green circle with the check mark in it. I
waited for about 20 seconds and nothing changed. I clicked the save button
up above and then a new pin showed up. I tried the pin and got the same
result.

**Comment by fuzeman** [comment]
> I changed the mode to Basic and used my username and password. After
> clicking login the gravatar came up like it was logged into the Plex
> account but there was not the green circle with the check mark in it. I
> waited for about 20 seconds and nothing changed.

The green checkmark only shows up once the plugin has updated your account and verified it's working, so you need to save the account changes before it will come up.

----------------

Could you do the following to get some more information on this issue:

1. In `Plex/Web -> Channels -> Trakt.tv -> Settings`, set all the "Logging" options to "DEBUG"
2. At http://trakt-for-plex.github.io/configuration, attempt another login with a Plex account that isn't working
3. Retrieve the latest plugin log file ([details here](https://github.com/trakt/Plex-Trakt-Scrobbler/wiki/Log-files)), and either post it here or email it to me@dgardiner.net

**Comment by montizzl3** [comment]
OK, here is the log.  Thanks for your help!

[com.plexapp.plugins.trakttv.log.zip](https://github.com/trakt-for-plex/configuration/files/376588/com.plexapp.plugins.trakttv.log.zip)

**Comment by montizzl3** [comment]
I figured out what my issue was and why it would not allow me to login with this user but it did allow other uers with no problem.  I had not yet shared any libraries on this server with that user.  I was going to get trakt set up to sync and then share out the libraries needed.  I'm sorry to waist your time on this.  As soon as I shared out a couple libraries with this user it logged in with no issue.

Thanks for all your hard work on this plug-in.  Truly top knoch.
